charts/opentelemetry-demo/values.schema.json

```diff
@@ -125,6 +125,9 @@
         },
         "podAnnotations": {
           "type": "object"
+        },
+        "nodeSelector": {
```

Comment:
Please add an empty `nodeSelector` property in the values.yaml, like `podAnnotations`, that helps users understand when this field would be used.

Reply:
makes sense, all done!